src/css/components/nav.scss

```diff
@@ -37,16 +37,24 @@
     &:hover {
       color: $color-primary;
     }
+    &[href *= ".io"] {
```

Comment:
Sure, we can address it in a refactor once we figure out what the use cases will look like in the future.